### ISSUE TITLE
Recommend `tockloader --flash-file` instead of `objcopy` hackery

### ISSUE THREAD

**Issue by lschuermann** [tock/tock#4199]
For some boards, we're currently pointing users to some `objcopy` hackery to embed applications in a kernel binary, like that of the Nucleo-F429ZI:

https://github.com/tock/tock/blob/10f7d59afab64dae2c68812278136267b5f9ce2e/boards/nucleo_f429zi/README.md?plain=1#L25-L39

I'm wondering whether it makes sense to guide users to these commands when we have the flash-file backend in Tockloader that can do all of this, and work well with multiple apps across both ARM and RISC-V. 

Should we update the READMEs to point to Tockloader instead? We will likely have to add support to Tockloader for some of those boards in the process.

_Originally posted by @lschuermann in https://github.com/tock/tock/issues/4191#issuecomment-2405086771_

**Comment by bradjc** [comment]
Yes we should because then tockloader can handle app placement and ordering.

But really we should add proper support in tockloader with autodetection.
